### PR TITLE
dashboard design and layout changes

### PR DESCRIPTION
**Pull request by bestworks221**
## Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).

### CODE REVIEW

File: resources/js/pages/dashboard.vue [hunk 1]
```diff
@@ -1,79 +1,275 @@
...
-                    </div>
-                    <div class="description-container">
-                        <p class="description">Offering discounts for better online a store can loyalty weapon into driving</p>
+            <div class="col-sm-12 col-md-4 my-sm-3 my-md-0 leaders-container">
+                <div class="dashboard-card">
+                    <div class="card-title-wrapper d-flex justify-content-between">
+                        <div class="card-details-wrapper">
+                            <h2>New Courses</h2>
+                        </div>
+                        <!-- <div class="card-btn-wrapper">
```
Comment (ri-bulatao) [inline resources/js/pages/dashboard.vue:223]: Remove Unused Codes

File: resources/js/pages/dashboard.vue [hunk 2]
```diff
@@ -94,7 +290,7 @@ export default {
     },
 
     mounted() {
-        this.initCalendly()
+        // this.initCalendly()
```
Comment (ri-bulatao) [inline resources/js/pages/dashboard.vue:293]: Remove Commented Codes

File: resources/views/spa.blade.php [hunk 3]
```diff
@@ -31,7 +31,7 @@
   </script>
 
 
-  {{-- <script src="{{ (str_starts_with($appJs, '//') ? 'http:' : '').$appJs }}"></script> --}}
+{{-- <script src="{{ (str_starts_with($appJs, '//') ? 'http:' : '').$appJs }}"></script> --}}
```
Comment (ri-bulatao) [inline resources/views/spa.blade.php:34]: Remove Commented Codes

**Review by ri-bulatao — CHANGES_REQUESTED** [review]

**Comment by ri-bulatao** [comment]
Please Add Screenshots/Recording for the UI on the User Management Module

File: resources/views/spa.blade.php (hunk 3 above)
Comment (bestworks221) [reply resources/views/spa.blade.php:34]: Commented code is the code we are using locally. to reflect our changes for the front end

File: resources/js/pages/dashboard.vue (hunk 1 above)
Comment (bestworks221) [reply resources/js/pages/dashboard.vue]: Done

File: resources/js/pages/dashboard.vue (hunk 2 above)
Comment (bestworks221) [reply resources/js/pages/dashboard.vue]: Done

**Comment by bestworks221** [comment]
> Please Add Screenshots/Recording for the UI on the Dashboard Module

https://prnt.sc/HMETTJMLKyeo
https://prnt.sc/CflltRtZd5f-

**Pull request merged**